### DISCUSSION TITLE
No Wi-Fi no local access ?

### DISCUSSION THREAD

**Discussion by gorper** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#597] [General]
Just wondering, if module not find associated wifi does it stop there or should it be possible to use buttons on display? AP is tuned off. Or am I experiencing something else.
It has happened three times when power is lost to pool pump and I do not get the project to work again when power is restored and of course the display is not responding. Only fix have been to rip out the complete project and connect display to pump and restart it. Then it is fine to power off and re install module and all is fine again. The blue led is not blinking when it happens. I can’t rule out that it is not issue finding Wi-Fi.
I run a pre 2021 6wire system with the v2B board with red lcc. It is running 414 release

**Comment by gorper** [comment]
I Give me a partial answer after some digging, #522 is describing pretty well what I experienced BUT. I had the AP manager tuned off and the device anyhow get stuck and give no local access from Display. But I must investigate more when I am where I have the pump to be sure.

**Reply by visualapproach** [reply]
I'm fixing it. Call you back when ready

**Reply by visualapproach** [reply]
Ready! (Beta) NTP sync was blocking the code. It should now work without wifi and without AP manager. If you need you can press POWER-LOCK-TIMER-POWER to reset wifi credentials and restart AP manager.

Thanks for your post!

**Reply by gorper** [reply]
Thanks and keep up the good work.
